README.maintainer.md

```diff
@@ -12,11 +12,10 @@ Following actions need to be taken to release a new stable version of BlueChi:
 
 Following files need to be updated to create a new stable release commit:
 
-* `RELEASE` variable needs to be updated in `build-scripts/build-srpm.sh` to contain a number instead of date and git
-  hash, which are used for snapshot builds. The number should contain `1` for the first build of the release, but it
-  may be increased if we need additional package rebuilds.
-* A new `%changelog` section in `bluechi.spec.in` should be added and it should contain information about the new package
-  release
+* In [build-scripts/version.sh](./build-scripts/version.sh) set `IS_RELEASE=true`. For the first release of that version
+`RELEASE=1` is used. This may be increased if additional package releases are needed.
+* A new `%changelog` section in [bluechi.spec.in](./bluechi.spec.in) should be added and it should contain information
+about the new package release
```

Comment:
Probably worth change a link to an example of the release commit as things changed significanlty from 0.1.0

Reply:
Good point, done :heavy_check_mark: